src/els_indexer.erl

```diff
@@ -71,24 +71,30 @@ index(#{uri := Uri, text := Text} = Document) ->
     {ok, [#{md5 := MD5}]} ->
       ok;
     _ ->
-      F = fun() ->
-              ok = els_dt_document:insert(Document),
-              Module = els_uri:module(Uri),
-              Specs  = els_dt_document:pois(Document, [spec]),
-              [els_dt_signatures:insert(#{ mfa  => {Module, F, A}
-                                         , tree => Tree
-                                         }) ||
-                #{id := {F, A}, data := Tree} <- Specs],
-              POIs  = els_dt_document:pois(Document, [ application
-                                                     , implicit_fun
-                                                     ]),
-              ok = els_dt_references:delete_by_uri(Uri),
-              [register_reference(Uri, POI) || POI <- POIs],
-              ok
-          end,
+      F = fun() -> do_index(Document) end,
       els_db:transaction(F)
   end.
 
+-spec do_index(els_dt_document:item()) -> ok.
+do_index(#{uri := Uri, id := Module, kind := Kind} = Document) ->
```

Comment:
nit: The `Id` is not necessarily a `Module`.

Reply:
Renamed the variable to `Id`.

> The Id is not necessarily a Module.

This was the case before the changes in this PR, right?